src/main.js

```diff
@@ -58,7 +110,10 @@ function handleSelection({ id, value }) {
 
   console.log('');
   // Run menu again, 'quit' is the only way to get out of loop
-  runOptionsMenu();
+
+  if (id != 'add') {
```

Comment:
Ahhh, you don't want the loop after you run add cmd?
Hmmm wonder if there's a better way of expressing this. Fine for now though.

Reply:
Oh I see what's happening, you're having async problems.

Reply:
You can remove this if

Reply:
Yeah, basically this is just a quick way to stop the CLI while the add book by isbn function does its thing. I put runOptionsMenu(); at the bottom of my function on line 74. If I take it out the CLI is on top of my function interface.

Reply:
Okay after all changes were complete, I'm now getting the problem of the CLI menu being overtop the ISBN entry field. SO when running: CLI appears, select add book,  function runs and it asks for ISBN overtop of the CLI quit line. Entering the ISBN and pressing enter works, but also enters "show book" (since it's selected). Adding this 'if statement' back in fixes the overlapping issue, but the the OptionsMenu doesn't loop after my ISBN function.

Am I making sense?

Reply:
<img width="250" alt="Screen Shot 2022-04-01 at 3 57 48 PM" src="https://user-images.githubusercontent.com/74270793/161333035-bfa33602-5d89-4183-b7dc-b7207290772d.png">

So here's a screen, I started the program and selected add book, then I typed 'ISBN ENTRY AREA IS HERE'

Reply:
Let me take a quick look at this

Reply:
Ok, I see what's going on...
You're using the 'readline' module in node which is meant to be used with streams and it has a callback API. It is a fairly deep topic in and of itself, but suffice it to say that the callback function you are writing in `rl.question` is not being `await`ed.

For this, you have a few solutions:

1. You can use a synchronous version of readline: https://www.npmjs.com/package/readline-sync
2. You can "promisify" the `rl.question` for that you can `await` the read in the terminal. 

For #2 I have made a sample of it working in the `enhance/more-bookz-readline-fix` branch.

More info:
- https://nodejs.org/dist/latest-v16.x/docs/api/readline.html
- https://stackoverflow.com/questions/43638105/how-to-get-synchronous-readline-or-simulate-it-using-async-in-nodejs